resources/js/components/tools/importer/ImporterUpdateState.vue

```diff
@@ -0,0 +1,119 @@
+<template>
+    <div class="importer-update-state d-flex card ">
+        <header class="card-header">
+            {{ t('main.importer.validation.title') }}
+        </header>
+        <div class="card-body ">
+            <div
+                class="alert d-flex align-items-center gap-4"
+                :class="activeOption.class"
+            >
+                <i :class="activeOption.icon" />
+                <span>
+                    {{ t(`main.importer.validation.${activeOption.text}`) }}
+                </span>
+            </div>
+
+            <div class="input-group">
+                <ImporterUpdateItem
+                    :value="create"
+                    icon="fas fa-fw fa-circle-plus"
+                    class="input-group-text flex-grow-1 text-success"
+                    :title="t('global.create')"
+                />
+                <ImporterUpdateItem
+                    :value="update"
+                    icon="fas fa-fw fa-circle-arrow-up"
+                    class="input-group-text flex-grow-1 text-warning"
+                    :title="t('global.update')"
+                />
+                <ImporterUpdateItem
+                    :value="conflict"
+                    icon="fas fa-fw fa-circle-exclamation"
+                    class="input-group-text flex-grow-1 text-danger"
+                    :title="t('global.conflict')"
+                />
+            </div>
+        </div>
+    </div>
+</template>
+
+<script>
+    import { computed } from 'vue';
+    import { useI18n } from 'vue-i18n';
+
+    import ImporterUpdateItem from './ImporterUpdateItem.vue';
+
+    export default {
+        components: {
+            ImporterUpdateItem
+        },
+        props: {
+            conflict: {
+                type: Number,
+                required: true
+            },
+            update: {
+                type: Number,
+                required: true
+            },
+            create: {
+                type: Number,
+                required: true
+            },
+
+        },
+        setup(props) {
+
+            const { t } = useI18n();
+
+            const options = computed(() => {
```

Comment:
is computed required here?

Reply:
Works without computed. Fixed in d4e5fdfd9084423e1e8fa01e8476322e424c328e